driver/bpf/Makefile

```diff
@@ -14,6 +14,8 @@ KERNELDIR ?= /lib/modules/$(shell uname -r)/build
 
 # DEBUG = -DBPF_DEBUG
 
+KBUILD_CPPFLAGS += -fno-stack-protector
```

Comment:
Makes sense, I'm just wondering if there is a reason why we are setting the `KBUILD_CPPFLAGS` variable rather than appending the `-fno-stack-protector` flag directly to the clang invocation, like all the other compiler flags we already have a few lines below?

Are we somehow expecting the kernel's main Makefile to process this flag explicitly and add some extra definitions depending on its presence? If not, I'd find it more clear and consistent to just append it to clang since we basically just use the kernel Makefile to generate the right `-I` paths for our explicit clang invocation, but that's just my opinion.

Reply:
Fair point. Fixed

Reply:
Thanks. LGTM.